editor/README.md

```diff
@@ -28,24 +27,19 @@ Serving depends on:
 
 To build the ord-schema python code, follow the instructions [here](https://github.com/Open-Reaction-Database/ord-schema/blob/main/README.md).
 
-For the protobuf compiler and Javascript runtime libraries, the editor currently requires HEAD protobuf (to get experimental "optional" declarations).
+Download the protobuf JS runtime as follows. At the time of writing, version
```

Comment:
We can probably just nuke all of this since it's handled by the Dockerfile, right?

Reply:
Even if the procedure is entirely scripted, I think documenting it still has value.

Also for the time being I continue to work in a manually crafted environment because it's faster and easier to iterate the JS. Nathan told me he does also.

Reply:
SGTM, thanks.